thoth/storages/graph/postgres.py

```diff
@@ -5231,11 +5219,6 @@ def sync_solver_result(self, document: dict, *, force: bool = False) -> None:
         os_name = solver_info["os_name"]
         os_version = solver_info["os_version"]
         python_version = solver_info["python_version"]
-        # Older solver documents did not provide platform explictly.
-        platform = document["result"].get("platform") or "linux-x86_64"
-
-        if not self.python_package_version_depends_on_platform_exists(platform=platform):
```

Comment:
I think we want to keep this check in any case, because when we will have more platforms we need to take care of that column. wdyt?

Reply:
oh you mean like lets leave the check, and decide in future if we wanna add that to the column.
understood. will get these back.

Reply:
Yes exactly, even if we don't have other platforms for now, better to keep it!